spec/searchBarSubMenu.component.spec.js

```diff
@@ -0,0 +1,112 @@
+let viewName = "TEST";
```

Comment:
@etgrieco if this is the same config as the controller could you create a fixture file for it instead and reuse in both tests?

Reply:
I agree, let me look into how to use a fixture